src/webview/helm-chart/app/helmListItem.tsx

```diff
@@ -104,6 +104,10 @@ function HelmChartListContent(props: HelmListItemProps) {
                 >
                     {props.selectedVersion.description}
                 </Typography>
+                <Typography
+                    variant='body2'
+                    sx={{ whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis' }}
+                >Repo: https://charts.openshift.io/</Typography>
```

Comment:
@msivasubramaniaan Let's make the URL as a hyperlink so that users can directly open it if needed.

Reply:
![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/cff874bd-a693-4087-a2b0-5018c109cb15)
Done